pkg/handlers/archive.go

```diff
@@ -222,6 +255,8 @@ type mimeType string
 const (
 	arMimeType  mimeType = "application/x-unix-archive"
 	rpmMimeType mimeType = "application/x-rpm"
+	machOType   mimeType = "application/x-mach-binary"
+	octetStream mimeType = "application/octet-stream"
```

Comment:
What about ELF or EXE files? Are those covered by `application/octet-stream`?

Reply:
Some of the reading i did they showed up as octet-stream, but i also read that they can be `application-x-elf`. Just to be safe, I added additional extensions to the list of known binary extensions. There might be a more robust way of determining if the file is a binary format.